addons/spreadsheet/static/tests/charts/model/odoo_chart_plugin.test.js

```diff
@@ -628,3 +628,104 @@ test("Remove odoo chart when sheet is deleted", async () => {
     model.dispatch("DELETE_SHEET", { sheetId });
     expect(model.getters.getOdooChartIds().length).toBe(0);
 });
+
+describe("See records when clicking on an element of a chart", async () => {
+    let sheetId, chartId, model;
```

Comment:
This is useless, declare them in each test with `const`
```suggestion
```

Reply:
Well, in fact, the first test of this describe was supposed to be a beforeEach defining this :D